### ISSUE TITLE
CI Build is failing (due to an oasis bug)

### ISSUE THREAD

**Issue by maz** [JHU-PL-Lab/big-bang#39]
The travis build is failing. It looks like it may be due to this bug: https://github.com/ocaml/opam-repository/issues/5172

So, to fix this, is it okay if I add

```
ocamlfind remove plugin-loader
ocamlfind remove userconf
ocamlfind remove oasis
```

to the `.travis-ci.sh` script? According to the issue that should fix it.

@zepalmer @leafac

**Comment by leafac** [comment]
Sure! Go ahead and try that :+1:

**Comment by maz** [comment]
Closed by #40
